userguide/tutorials/getting_started.adoc

```diff
@@ -284,58 +284,136 @@ The next step is to configure Kill Bill. For this, you need to follow the steps
 . Start Tomcat using the script *TOMCAT_HOME/bin/startup.sh* or *TOMCAT_HOME/bin/startup.bat*
 (Depending on your operating system)
 
-. Verify that there are no errors in the Tomcat logs at  *TOMCAT_HOME/logs/catalina.out*.
+. Verify that there are no errors in the Tomcat logs at  *TOMCAT_HOME/logs/catalina.log*
 
 . Verify that there are no errors in the Kill Bill logs on the console and that the logs display a line which states that  *Kill Bill server has started*
 
 . Open a browser and type http://localhost:8080/index.html. If Kill Bill is configured properly, it should display the following page:
 
 image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/img/getting-started/killbill-home.png[align=center]
 
+==== Database Configuration (Kaui)
 
+In addition to the Kill Bill database, you will also need to configure the Kaui database. For this, you need to follow the steps given below:
+
+. Create a database. In MySQL, you can run the following commands to create a database called *kaui*: 
+
+    create database kaui;
+   
+
+. Run the Kaui DDL https://github.com/killbill/killbill-admin-ui/blob/master/db/ddl.sql[here].
+
+. You should have a database called *kaui* (Or whatever name you have specified in step 1) with the following tables:
+
+ kaui_users
+ kaui_tenants
+ kaui_allowed_users
+ kaui_allowed_user_tenants
+
+
+==== Kaui Configuration
+
+Finally, Kaui needs to be configured. For this, you need to follow the steps given below:
+
+. Download the current version of the Kaui war from https://search.maven.org/search?q=a:kaui-standalone[Maven Central].
+
+. Rename the war file downloaded above to `kaui.war` 
+
+. Ensure that Tomcat server is stopped.
+
+. Place the downloaded war file at *TOMCAT_HOME/webapps*. So, *webapps* folder should look like this:
+
+ $TOMCAT_HOME/webapps
+     --ROOT.war
+     --kaui.war
+
+. Open *TOMCAT_HOME/conf/catalina.properties file*. Add the following database properties  related to Kaui at the end of this file (Use appropriate values as per your database):
+
+ kaui.url=http://127.0.0.1:8080
+ kaui.db.url=jdbc:mysql://127.0.0.1:3306/kaui
+ kaui.db.username=killbill
+ kaui.db.password=killbill
+
+. Start Tomcat using the script *TOMCAT_HOME/bin/startup.sh* or *TOMCAT_HOME/bin/startup.bat*
+(Depending on your operating system)
+
+. Verify that there are no errors in the Tomcat logs at  *TOMCAT_HOME/logs/catalina.log*
+
+. Verify that there are no errors in the Kill Bill logs on the console and that the logs display a line which states that  *Kill Bill server has started*
+
+. Open a browser and type http://localhost:8080/kaui This should display the following sign in page: image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/img/kaui_sign_in.png[align=center]
+
+. Sign in with `admin/password`. This should display the following page:
+
+image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/img/kaui_after_sign_in.png[align=center]
```

Comment:
@learnjavawithreshma I just deployed the changes but it looks like there is a typo in both image urls (missing `getting-started/` in the urls).

Check https://docs.killbill.io/latest/getting_started.html#_tomcat

Reply:
@pierre, Sorry, my bad, fixed this now!